src/schema/v2/artist/createArtistMutation.ts

```diff
@@ -0,0 +1,103 @@
+import {
+  GraphQLBoolean,
+  GraphQLNonNull,
+  GraphQLString,
+  GraphQLUnionType,
+} from "graphql"
+import { GraphQLObjectType } from "graphql"
+import { mutationWithClientMutationId } from "graphql-relay"
+import {
+  formatGravityError,
+  GravityMutationErrorType,
+} from "lib/gravityErrorHandler"
+import { snakeCase } from "lodash"
+import { ResolverContext } from "types/graphql"
+import { ArtistType } from "./index"
+
+interface CreateArtistMutationInput {
+  birthday: string
+  deathday: string
+  displayName: string
+  first: string
+  isPersonalArtist: boolean
+  last: string
+  middle: string
+  nationality: string
+}
+
+const CreateArtistSuccessType = new GraphQLObjectType<any, ResolverContext>({
+  name: "CreateArtistSuccess",
+  isTypeOf: (data) => {
+    return data._type !== "GravityMutationError"
```

Comment:
This is a tiny bit indirect. When there's not an error, `data` will be the return of the `mutateAndGetPayload` method, so the result of `artistLoader`. Which won't even _have_ a `_type` field set.

So while this is technically true, it feels indirect (although the checking for error type is good bc `_type: 'GravityMutationError'` is injected into those responses).

I'd recommend instead checking for something about the `artistLoader` response directly, like `isTypeOf: (data) => data.id` for instance.